LabApiUtilities/src/test/com/microsoft/identity/labapi/utilities/client/LabClientTest.java

```diff
@@ -104,4 +133,76 @@ public void canCreateBasicTempUser() {
         }
     }
 
+    @Test
+    public void canCreateMAMCATempUser() {
+        final LabApiAuthenticationClient authenticationClient = new LabApiAuthenticationClient(
+                TestBuildConfig.LAB_CLIENT_SECRET
+        );
+
+        final LabClient labClient = new LabClient(authenticationClient);
+
+        try {
+            final ILabAccount labAccount = labClient.createTempAccount(TempUserType.MAM_CA);
+            Assert.assertNotNull(labAccount);
+            Assert.assertNotNull(labAccount.getUsername());
+            Assert.assertNotNull(labAccount.getPassword());
+            Assert.assertNotNull(labAccount.getUserType());
+            Assert.assertTrue(labAccount.getUsername().toLowerCase().contains("msidlab4"));
+        } catch (final LabApiException e) {
+            throw new AssertionError(e);
+        }
+    }
+
+    @Test
+    @Ignore
+    public void canResetPassword() {
+        final LabApiAuthenticationClient authenticationClient = new LabApiAuthenticationClient(
+                TestBuildConfig.LAB_CLIENT_SECRET
+        );
+
+        final LabClient labClient = new LabClient(authenticationClient);
+
+        try {
+            final ILabAccount labAccount = labClient.createTempAccount(TempUserType.BASIC);
+            Thread.sleep(timeout);
+            Assert.assertTrue(labClient.resetPassword(labAccount.getUsername(), 2));
+        } catch (final LabApiException | InterruptedException e) {
+            throw new AssertionError(e);
+        }
+    }
+
+    @Test
+    public void canEnablePolicy() {
+        final LabApiAuthenticationClient authenticationClient = new LabApiAuthenticationClient(
+                TestBuildConfig.LAB_CLIENT_SECRET
+        );
+
+        final LabClient labClient = new LabClient(authenticationClient);
+
+        try {
+            final ILabAccount labAccount = labClient.createTempAccount(TempUserType.BASIC);
+            Thread.sleep(timeout);
```

Comment:
This logic of creating temp and Waiting could probably move into a helper method and then invoke that helper method in all the places. This way if you need to make changes to this logic in the future then you make it in that one helper method.

Reply:
Done